ansible/catch.yml

```diff
@@ -12,6 +12,6 @@
 
     - name: "Download"
       become: yes
-      shell: "wget https://raw.githubusercontent.com/catchorg/Catch2/master/single_include/catch2/catch.hpp"
+      shell: "wget https://raw.githubusercontent.com/catchorg/Catch2/v2.x/single_include/catch2/catch.hpp"
```

Comment:
I needed this to compile the tests properly as the `master` branch was broken. It's a temporary workaround before #322 is merged. Shouldn't create too many conflicts.